Binance.Net/Objects/BinanceFuturesPosition.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using CryptoExchange.Net.Converters;
+
+namespace Binance.Net.Objects
+{
+    /// <summary>
+    /// Information about an account
+    /// </summary>
+    public class BinanceFuturesPosition
+    {
+        /// <summary>
+        /// The entry price of the position
+        /// </summary>
+        public decimal EntryPrice { get; set; }
+        /// <summary>
+        /// Type of margin used for the position
+        /// </summary>
+        public FuturesMarginType MarginType { get; set; }
```

Comment:
Doesn't this need a specified converter?

Reply:
I added the converters with the first commit. Sorry for not being clear with that.

https://github.com/JKorf/Binance.Net/pull/279/commits/b72a675c02f96dc9f36d73e15c9ed86e9ade3e4f#diff-53cef88717420750dd95c29891fe67bc